[May18] Surface Magnetism + Spatial Processing example scene - improvement

cre8ivepark opened the pull request:
Overview
---
- Added transparent surface to make it easier to see the interaction with physical surfaces.
- Assigned WireframeBlue material to the example scene. (it was not assigned)
- [Updated landing page README for the discoverability
](https://github.com/cre8ivepark/MixedRealityToolkit-Unity/blob/may18_dev_SurfaceMagnetism/README.md)
![2018-05-17 13_28_16-mixedrealitytoolkit-unity_readme md at dev_working_branch_surfacemagnetism cre](https://user-images.githubusercontent.com/13754172/40202246-3ce07062-59d6-11e8-8f3b-fb279b719db9.png)

Changes
---
- Updates to #2114

StephenHodgson commented on ProjectSettings/TagManager.asset:
```diff
@@ -13,7 +13,7 @@ TagManager:
   - 
   - 
-  - 
+  - SpatialProcessingPlane
```
Not sure we should be adding anything to the tag manager manually.

When people import this example, tags are generally not included.

StephenHodgson replied:
It'll probably be better to hard-code this in the example code.

cre8ivepark replied:
I didn't want to modify the code in SurfaceMeshesToPlanes.cs where it assigns dummy layer 31
![2018-05-17 12_43_53-mrtk - microsoft visual studio](https://user-images.githubusercontent.com/13754172/40200015-082a65ea-59d0-11e8-88cd-4ce9757dbdf4.png)

Solver Surface Magnetism script uses the layer through this drop down.
![2018-05-17 12_42_19-unity 2017 2 1p2 64bit - spatialprocessingsurfacemagnetism unity - mrtk - univ](https://user-images.githubusercontent.com/13754172/40199932-cac9e52c-59cf-11e8-8286-3b8aa764d90f.png)

StephenHodgson replied:
It was just an example of how to implement a dialog to prompt devs to add the layer in their project.

cre8ivepark replied:
Sorry I just read #1443. In this case, I think we can use layer 31:Spatial Mapping and add instruction to use Configure > Apply Mixed Reality Project Settings.
Thank you!

![2018-05-17 12_49_47-unity 2017 2 1p2 64bit - spatialprocessingsurfacemagnetism unity - mrtk - univ](https://user-images.githubusercontent.com/13754172/40200329-dc8cc4a4-59d0-11e8-8f23-05e0d06bdbbe.png)

![2018-05-17 12_49_40-unity 2017 2 1p2 64bit - spatialprocessingsurfacemagnetism unity - mrtk - univ](https://user-images.githubusercontent.com/13754172/40200330-dca1a7fc-59d0-11e8-988c-9e0339ad3d71.png)

cre8ivepark commented:
- Removed TagManager.asset
- Updated README to use MRTK's Configure menu to add the layer 31:SpatialMapping
- Updated scene to use the layer 31:SpatialMapping

https://github.com/cre8ivepark/MixedRealityToolkit-Unity/blob/may18_dev_SurfaceMagnetism/Assets/HoloToolkit-Examples/SpatialMapping/SpatialProcessingSurfaceMagnetism.md

davidkline-ms commented on Assets/HoloToolkit-Examples/SpatialMapping/SpatialProcessingSurfaceMagnetism.md:
```diff
@@ -12,17 +12,11 @@ https://gfycat.com/PitifulRareKodiakbear
 
-2. In SpatialMapping prefab, make sure to set the Physics Layer to 31.
-<img src="/External/ReadMeImages/MRTK_SurfaceMagnetism5.png" width="550">
+2. In SpatialMapping prefab, make sure to set the Physics Layer to 31. <br/><img src="/External/ReadMeImages/MRTK_SurfaceMagnetism5.png" width="550">
```
This is the default within the prefab. I would reword step 2 to say "make sure the Physics Layer is set to 31"

davidkline-ms commented on Assets/HoloToolkit-Examples/SpatialMapping/SpatialProcessingSurfaceMagnetism.md:
```diff
@@ -12,17 +12,11 @@ https://gfycat.com/PitifulRareKodiakbear
 
-3. Assign SolverSurfaceMagnetism script to an object. In Magnetic Surface option, select the new layer **SpatialMapping** that you created. 
-<img src="/External/ReadMeImages/MRTK_SurfaceMagnetism6.png" width="550">
+3. Assign SolverSurfaceMagnetism script to an object. In Magnetic Surface option, select the new layer **SpatialMapping** that you created.<br/><img src="/External/ReadMeImages/MRTK_SurfaceMagnetism6.png" width="550">
```
"select the SpatialMapping layer." they would not have created it

cre8ivepark replied:
Updated, Thanks!
